flow/scripts/floorplan.tcl

```diff
@@ -123,11 +121,11 @@ if { [info exist ::env(RESYNTH_AREA_RECOVER)] && $::env(RESYNTH_AREA_RECOVER) ==
   write_verilog $::env(RESULTS_DIR)/2_pre_abc.v
 
   set tielo_cell_name [lindex $env(TIELO_CELL_AND_PORT) 0]
-  set tielo_lib_name [get_name [get_property [get_lib_cell $tielo_cell_name] library]]
+  set tielo_lib_name [get_name [get_property [lindex [get_lib_cell $tielo_cell_name] 0] library]]
```

Comment:
Why is this change needed?

Reply:
I'll need to double check to see if get_lib_cell still returns multiple cells when multiple corners are loaded, but it's to ensure that `get_property` only gets a single argument instead of a list.

Reply:
>>> set tielo_cell_name [lindex $env(TIELO_CELL_AND_PORT) 0]
TIELOW
>>> get_lib_cell $tielo_cell_name
_00fcc20f00000000_p_LibertyCell _20a0151a00000000_p_LibertyCell _00d3612400000000_p_LibertyCell

Still returns 3 so lindex just gives one of them.